torchsde/_brownian/brownian_interval.py

```diff
@@ -477,23 +477,18 @@ def __call__(self, ta, tb=None, return_U=False, return_A=False):
             if len(intervals) > 1:
                 # If we have multiple intervals then add up their increments and Levy areas.
 
-                # Clone to avoid modifying the W, H, A that may exist in the cache
-                W = W.clone()
-                if self.have_H:
```

Comment:
Excellent! I was hoping that the clones be removed at some point.

Reply:
What's there isn't any more efficient as the operations are now out of place instead of in-place. Just because it's marginally neater to look at, really!